tests/printer/expr/expected/binary.res.txt

```diff
@@ -496,16 +492,13 @@ let aggregateTotal = (forecast, ~audienceType) =>
 React.useEffect4(() => {
   switch (context.library, context.account) {
   | (Some(library), Some(account)) =>
-    library.getBalance(. account)
-    ->Promise.Js.catch(_ => {Promise.resolved(None)})
-    ->Promise.get(newBalance => {
-      dispatch(
-        LoadAddress(
-          account,
-          newBalance->Belt.Option.flatMap(balance => Eth.make(balance.toString(.))),
-        ),
-      )
-    })
+    library.getBalance(. account)->Promise.Js.catch(_ => {Promise.resolved(None)})->Promise.get(
```

Comment:
Promises really profit from per-line pipe formatting. But I guess this might not even be relevant anymore for the "next big feature"?

Reply:
This seems indeed like a "regression" compared to the previous formatted code